content/docs/1.6.0/deploy/install/install-with-flux.md

```diff
@@ -42,9 +43,10 @@ weight: 11
     flux create helmrelease longhorn-release \
       --chart=longhorn \
       --source=HelmRepository/longhorn-repo \
-      --chart-version=1.6.0 \
+      --chart-version=v1.6.0 \
```

Comment:
Replace `v1.6.0` with `v{{< current-version >}}`

Reply:
Refined. Thank you!